serverless.yml

```diff
@@ -33,6 +33,7 @@ provider:
     obfuscationSpinShapefiles: ${self:custom.variables.obfuscationSpinShapefiles}
 
     orderDelaySeconds: ${self:custom.variables.orderDelaySeconds}
+    useCache: ${self:custom.variables.useCache}
```

Comment:
It seems silly to have this config versus just having `npm run start` spin up the docker container automatically.